prisma/seed.ts

```diff
@@ -10,6 +10,7 @@ const prisma = new PrismaClient();
  */
 const deleteUsers = async () => {
   await prisma.user.deleteMany({});
+  await prisma.invitation.deleteMany({});
```

Comment:
Are you sure we need this line? I know the delete behavior is weird in Prisma, but I recently noticed that if you delete a ``User`` in ``prisma studio``, it cascades and deletes the connected ``Account`` and ``Session`` entry.

Reply:
No harm leaving it there. Unless @owensimpson wants to look into the user delete behavior and validate the need for the extra call.